packages/wallets/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC, useMemo } from 'react';
+import { useAvailableMT5Accounts } from '@deriv/api';
+import { WalletText } from '../../../../components/Base/WalletText';
+import { useModal } from '../../../../components/ModalProvider';
+import { JurisdictionCard } from './JurisdictionCard';
+import './JurisdictionScreen.scss';
+
+type TJurisdictionScreenProps = {
+    selectedJurisdiction: string;
+    setSelectedJurisdiction: React.Dispatch<React.SetStateAction<string>>;
+};
+
+const JurisdictionScreen: FC<TJurisdictionScreenProps> = ({ selectedJurisdiction, setSelectedJurisdiction }) => {
+    const { modalState } = useModal();
+    const { data, isLoading } = useAvailableMT5Accounts();
+    const jurisdictions = useMemo(
+        () =>
+            data?.filter(account => account.market_type === modalState?.marketType).map(account => account.shortcode) ||
+            [],
+        [data, modalState?.marketType]
+    );
+
+    if (isLoading) return <WalletText>Loading...</WalletText>;
+
+    return (
+        <div className='wallets-jurisdiction-screen'>
+            <div className='wallets-jurisdiction-screen__cards'>
+                {jurisdictions.map(jurisdiction => (
+                    <JurisdictionCard
+                        isSelected={selectedJurisdiction === jurisdiction}
+                        jurisdiction={jurisdiction || 'bvi'}
```

Comment:
I don't think we should be hardcoding the jurisdiction value `'bvi'`, btw is this required ?

Reply:
'bvi' as a default value should be fine if jurisdiction value from the hook is undefined, I didn't put 'svg' because they might dissolve that